contrib/win32/openssh/OpenSSHTestHelper.psm1

```diff
@@ -506,13 +506,18 @@ function Get-UnitTestDirectory
     Run OpenSSH pester tests.
 #>
 function Invoke-OpenSSHE2ETest
-{     
+{
+    [CmdletBinding()]
+    param
+    (
+        [ValidateSet('CI', 'Scenario')]
```

Comment:
It is the priority of the tests. CI is continuous integration test which is basic tests required to run for each build. While Scenario is the scenario tests which may be run less frequent.